StardewArchipelago/GameModifications/RandomizedLogicPatcher.cs

```
@@ -87,6 +87,7 @@ public RandomizedLogicPatcher(LogHandler logger, IModHelper modHelper, ModConfig
             OutOfLogicInjections.Initialize(logger, archipelago, stardewItemManager);
             EmptyHandInjections.Initialize(logger, archipelago, stardewItemManager);
             DebugPatchInjections.Initialize(logger, archipelago);
+            BundleMenuInjection.setupLogging(logger);

```

Wow, this was never called before? So any error in that transpiler was just swallowed? Interesting...

Did you run into such errors, and that's why you needed the logger fixed?

While you're there, would you mind renaming it to `Initialize()` for consistency with the rest?

---

Yeah, that seemed to be a bit of an oversight! 

I did run into errors thrown in VS that were supposed to be logged, hence why I needed the logger fixed!

Will rename to `Initialize()`, it makes sense